src/build/config/config.yaml

```diff
@@ -215,6 +215,10 @@ optionGroup:
 #      defined for the option command override, then they override command roles defined for the option (rule 2) above)
 #      and all roles defined for the command (rule 1) above).
 #
+# bool-like:
+#   Used to convert a boolean to a string or string-id type. It can been used as a boolean (e.g. --option or option=y) and the
```

Comment:
"it can be"?

Reply:
I think it is just as good as "it may be".

Reply:
I was just nitpicking about the "it can be**en**" ;-)

Reply:
Oh, I see it *now*! Fixed in cff6695.